src/status_im2/contexts/syncing/sheets/sync_device_notice/views.cljs

```diff
@@ -0,0 +1,54 @@
+(ns status-im2.contexts.syncing.sheets.sync-device-notice.views
+  (:require
+   [quo.react-native :as rn]
+   [status-im2.contexts.syncing.sheets.sync-device-notice.styles :as styles]
+   [status-im2.contexts.syncing.sheets.enter-password.views :as enter-password]
+   [utils.re-frame :as re-frame]
+   [quo2.core :as quo]
+   [i18n.i18n :as i18n]))
+
+(defn views []
+  [:<>
+   [rn/view {:style styles/sync-devices-header}
+    [rn/image {:source (js/require "../resources/images/ui/sync-new-device-cover-background.png")
```

Comment:
i believe you have it requered already in resources?

Reply:
yes but resources is in status-im namespace and this seemed to be the easiest way of not importing the resources namespace into code that is in status-im2 namespace. I could remove the js/require from that namespace since it's not used here.

Reply:
removed the require from resources